app/src/main/java/edu/uc/clemenks/keepingupwithschool/ui/main/MainFragment.kt

```diff
@@ -20,12 +20,8 @@ import kotlinx.android.synthetic.main.main_fragment.*
 
 class MainFragment : Fragment() {
 
-    val CAMERA_PERMISSION_REQUEST_CODE = 1999
-    val CAMERA_REQUEST_CODE = 1998
-
-    companion object {
-        fun newInstance() = MainFragment()
-    }
+    private val CAMERA_PERMISSION_REQUEST_CODE = 1999
+    private val CAMERA_REQUEST_CODE = 1998
```

Comment:
I changed the visibilty modifiers based on the suggestion here: https://kotlinlang.org/docs/reference/visibility-modifiers.html

Reply:
Good idea... and nice citation.